platform/darwin/src/MGLStyle.h

```diff
@@ -177,6 +177,14 @@ static const NSInteger MGLStyleDefaultVersion = 9;
 /**
  Returns a layer that conforms to `MGLStyleLayer` if any layer with the given
  identifier was found.
+ 
+ @note Layer identifiers are not guaranteed to exist across styles or different
+    versions of the same style. Applications that use this API must set the style 
```

Comment:
s/must/must first/

I think that could be a point of confusion for developers.